clients/iOS/Grapevine/Grapevine/LoginViewController.swift

```diff
@@ -81,7 +81,7 @@ class LoginViewController: UIViewController {
                     if response.1 != nil {
                         print("debug response printing")
                         debugPrint(response)
-                        if response.1?.statusCode == 200 {
+                        if response.1?.statusCode == 201 {
```

Comment:
Any way to use destructuring rather than messing with `response.1` and `response.2`?